src/Surfnet/ServiceProviderDashboard/Domain/Entity/Service.php

```diff
@@ -140,7 +140,7 @@ class Service
     /**
      * @var \Doctrine\Common\Collections\ArrayCollection
      *
-     * @ORM\OneToMany(targetEntity="Entity", mappedBy="service", cascade={"persist"}, orphanRemoval=true)
+     * @ORM\OneToMany(targetEntity="Entity", mappedBy="service", cascade={"persist"})
```

Comment:
Did you remove this because the removal logic is also explicitly added in code in the service remove command handler?

I think this should not be part of this PR, but maybe I'm overlooking something here..

Reply:
@pablothedude explained orphan removal is no longer required when using the transactions. Orphan removal caused cleanup of orphaned entities that later where tried to be removed by the cascading delete option.